ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.js

```diff
@@ -76,4 +79,174 @@ describe('Workload CVE overview page tests', () => {
+            const riskPriorityHeader = 'th:contains("Risk priority")';
+            // eslint-disable-next-line no-unused-vars
+            const cvesBySeverityHeader = 'th:contains("CVEs by severity")';
+            const prioritizeByNamespaceButton = 'button:contains("Prioritize by namespace view")';
+            const defaultFiltersButton = 'button:contains("Default filters")';
+
+            function assertCveElementsArePresent() {
+                // TODO
+                // These cannot be relied on in CI until the table is refactored to use
+                // the new table component that always renders the header
+                // cy.get(riskPriorityHeader);
+                // cy.get(cvesBySeverityHeader);
+                cy.get(prioritizeByNamespaceButton);
+                cy.get(defaultFiltersButton);
+                cy.get(selectors.severityDropdown);
+                cy.get(selectors.fixabilityDropdown);
+            }
+
+            function assertCveElementsAreNotPresent() {
+                // TODO
+                // These cannot be relied on in CI until the table is refactored to use
+                // the new table component that always renders the header
+                // cy.get(riskPriorityHeader).should('not.exist');
+                // cy.get(cvesBySeverityHeader).should('not.exist');
+                cy.get(prioritizeByNamespaceButton).should('not.exist');
+                cy.get(defaultFiltersButton).should('not.exist');
+                cy.get(selectors.severityDropdown).should('not.exist');
+                cy.get(selectors.fixabilityDropdown).should('not.exist');
+            }
+
+            // Visit the Images tab
+            interactAndWaitForResponses(() => {
+                selectEntityTab('Image');
+            }, imageListRouteMatcherMap);
+
+            assertCveElementsArePresent();
+
+            // Visit the Images tab
+            interactAndWaitForResponses(() => {
+                selectEntityTab('Deployment');
+            }, deploymentListRouteMatcherMap);
+
+            assertCveElementsArePresent();
+
+            // Switch to the "without cves" view, we should stay on the deployments tab
+            interactAndWaitForResponses(() => {
+                changeObservedCveViewingMode('Images without vulnerabilities');
+            }, deploymentListRouteMatcherMap);
+
+            assertCveElementsAreNotPresent();
+
+            // Visit the Images tab
+            interactAndWaitForResponses(() => {
+                selectEntityTab('Image');
+            }, imageListRouteMatcherMap);
+
+            assertCveElementsAreNotPresent();
+        });
+
+        it('should apply the correct filters when switching between "with cves" and "without cves" views', () => {
+            // Since we want to test the behavior of the default filters with the two cve views, we
+            // do not clear them by default in this case
+            visitWorkloadCveOverview({ clearFiltersOnVisit: false });
```

Comment:
oh do we not want to clear them when switching between the `with CVEs` and `without CVEs` views? cc: @alanonthegit